ilc/client/GuardManager.js

```diff
@@ -34,10 +34,12 @@ export default class GuardManager {
                     route: {
                         meta: route.meta,
                         url: route.reqUrl,
+                        hostname: window.location.hostname,
```

Comment:
it's minor but I think it's a little bit better to use `host` instead of `hostname`. what do you think?

Reply:
Nope :) http://bl.ocks.org/abernier/3070589 I would work inconsistently with SSR then.